variables.tf

```diff
@@ -101,3 +101,9 @@ variable "extra_accounts" {
   type        = list(string)
   default     = []
 }
+
+variable "ssl_secret_key" {
```

Comment:
The name of the variable and description is a bit misleading or at least confusing because it's not about a "secret key". That being said, I also have some trouble coming up with a better name...